drivers/include/net/netdev2.h

```diff
@@ -74,6 +74,9 @@ typedef enum {
     NETDEV2_EVENT_RX_COMPLETE,      /**< finished receiving a packet */
     NETDEV2_EVENT_TX_STARTED,       /**< started to transfer a packet */
     NETDEV2_EVENT_TX_COMPLETE,      /**< finished transferring packet */
+#ifdef MODULE_OPENTHREAD
+    NETDEV2_EVENT_TX_COMPLETE_DATA_PENDING, /**< finished transferring packet and has data pending flag **/
```

Comment:
Yes, but in that process you also [throw together the `trac_status` `TX_SUCCESS_DATA_PENDING` and `TX_SUCCESS` into the event `TX_COMPLETE`](https://github.com/RIOT-OS/RIOT/pull/3729/files#diff-148e890332a08fcca786b1e362e7f970R727). Can you remember why?